README.md

```diff
@@ -117,7 +114,7 @@ NOTE, Ideally, we could provision wordpress-trunk into VVV via https://github.co
     1. Note, if xdebug is enabled your tests will run VERY slowly. See https://varyingvagrantvagrants.org/docs/en-US/references/xdebug/ Only enable xdebug while testing if you wish to step-through debug your tests.
     ```
     $ vagrant ssh
-    You can add these variables to your bash, but make sure they are at the bottom of ~/.bash_alias as not to conflict with vvv's settings.
+    You can add these variables to your bash, but make sure they are at the bottom of ~/.bash_profile as not to conflict with vvv's settings.
```

Comment:
.bash_aliases did not work when i ran the tests in vscode but bash_profile does. 🤷‍♀️